draft-ietf-gnap-core-protocol.md

```diff
@@ -5458,6 +5472,184 @@ This specification requests registration of the following scheme in the
 * Authentication Scheme Name: `GNAP`
 * Reference: {{use-access-token}} of {{&SELF}}
 
+## Media Type Registration
+
+This section requests registration of the following media types {{RFC2046}} in
+the "Media Types" registry {{IANA.MediaTypes}} in the manner described
+in {{RFC6838}}.
+
+To indicate that the content is a GNAP request message to be bound with a JOSE mechanism:
+
+* Type name: application
+* Subtype name: gnap-binding
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: binary
+* Security considerations: See {{security}} of {{&SELF}}
+* Interoperability considerations: n/a
+* Published specification: {{&SELF}}
+* Applications that use this media type: GNAP
+* Fragment identifier considerations: n/a
+* Additional information:
+  * Magic number(s): n/a
+  * File extension(s): n/a
+  * Macintosh file type code(s): n/a
+* Person & email address to contact for further information: IETF GNAP Working Group, txauth@ietf.org
+* Intended usage: COMMON
+* Restrictions on usage: none
+* Author: IETF GNAP Working Group, txauth@ietf.org
+* Change Controller: IETF
+* Provisional registration?  No
+
+
+To indicate that the content is a GNAP token rotation message to be bound with a JOSE mechanism:
+
+* Type name: application
+* Subtype name: gnap-binding-rotation
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: binary
+* Security considerations: See {{security}} of {{&SELF}}
+* Interoperability considerations: n/a
+* Published specification: {{&SELF}}
+* Applications that use this media type: GNAP
+* Fragment identifier considerations: n/a
+* Additional information:
+  * Magic number(s): n/a
+  * File extension(s): n/a
+  * Macintosh file type code(s): n/a
+* Person & email address to contact for further information: IETF GNAP Working Group, txauth@ietf.org
+* Intended usage: COMMON
+* Restrictions on usage: none
+* Author: IETF GNAP Working Group, txauth@ietf.org
+* Change Controller: IETF
+* Provisional registration?  No
+
+
+To indicate that the content is a GNAP message to be bound with a detached JWS mechanism:
+
+* Type name: application
+* Subtype name: gnap-binding+jwsd
```

Comment:
you don't have examples using this anymore, will it be clear how it is used?

Reply:
Yes we do: https://deploy-preview-533--gnap-core-protocol-editors-draft.netlify.app/draft-ietf-gnap-core-protocol.html#name-detached-jws

Reply:
```suggestion
* Subtype name: gnap-binding+jws
```